shell/platform/android/android_external_texture_gl.cc

```diff
@@ -68,6 +68,19 @@ void AndroidExternalTextureGL::Paint(SkCanvas& canvas,
   }
 }
 
+// The bounds we set for the canvas are post composition.
+// To fill the canvas we need to ensure that the transformation matrix
+// on the `SurfaceTexture` will be scaled to fill. We rescale and preseve
+// the scaled aspect ratio.
+void scaleToFill(float& scaleX, float& scaleY) {
```

Comment:
The naming convention would make this `SkSize ScaleToFill(SkSize size)`. Also, the in-out params are not frequently used. Please avoid them unless necessary.

Reply:
Done!